modules/nf-core/purgedups/splitfa/purgedups-splitfa.diff

```diff
@@ -0,0 +1,29 @@
+Changes in module 'nf-core/purgedups/splitfa'
+--- modules/nf-core/purgedups/splitfa/main.nf
++++ modules/nf-core/purgedups/splitfa/main.nf
+@@ -11,8 +11,9 @@
+     tuple val(meta), path(assembly)
+ 
+     output:
+-    tuple val(meta), path("*.split.fasta.gz"), emit: split_fasta
+-    path "versions.yml"                      , emit: versions
++    tuple val(meta), path("*.split.fasta.gz") , emit: split_fasta
++    tuple val(meta), path("*.merged.fasta.gz"), emit: merged_fasta
++    path "versions.yml"                       , emit: versions
+ 
+     when:
+     task.ext.when == null || task.ext.when
+@@ -20,8 +21,10 @@
+     script:
+     def args = task.ext.args ?: ''
+     def prefix = task.ext.prefix ?: "${meta.id}"
++    def useGzip = !( assembly instanceof List ? assembly.every{ it.name.endsWith(".gz") } : assembly.name.endsWith(".gz") )
+     """
+-    split_fa $args $assembly | gzip -c > ${prefix}.split.fasta.gz
++    ${useGzip ? 'gzip -c' : 'cat'} $assembly > ${prefix}.merged.fasta.gz
++    split_fa $args ${prefix}.merged.fasta.gz | gzip -c > ${prefix}.split.fasta.gz
```

Comment:
Any plans to add that back to nf-core/modules?

Reply:
Yes, eventually.